xpdacq/beamtime.py

```diff
@@ -150,17 +152,18 @@ def tseries(dets, exposure, delay, num, *, md = None):
 
 
 def _nstep(start, stop, step_size):
-    ''' helper function to compute number of steps and step_size
-    '''
+    """ helper function to compute number of steps and step_size
+    """
     requested_nsteps = abs((start - stop) / step_size)
 
-    computed_nsteps = int(requested_nsteps)+1 # round down for finer step size
+    computed_nsteps = int(
+        requested_nsteps) + 1  # round down for finer step size
```

Comment:
It wasn't less than 72 characters when the spaces were added between the addition.